lte/gateway/release/BUILD.bazel

```diff
@@ -73,6 +78,112 @@ pkg_deb(
-    package_file_name = "{fname}.deb".format(fname = FILE_NAME),
+    package_file_name = "{fname}.deb".format(fname = SCTPD_FILE_NAME),
+    version = VERSION,
+)
+
+### MAGMA BUILD
+
+pkg_filegroup(
+    name = "magma_service_definitions",
+    srcs = [
+        "//lte/gateway/deploy/roles/magma/files/systemd:magma_lte_service_definitions",
+        "//orc8r/tools/ansible/roles/fluent_bit/files:magma_fluent_bit_service_definition",
+        "//orc8r/tools/ansible/roles/gateway_services/files:magma_orc8r_service_definitions",
+    ],
+    prefix = "/etc/systemd/system",
+)
+
+pkg_filegroup(
+    name = "magma_ebpf",
+    srcs = [
+        "//lte/gateway/python/magma/kernsnoopd/ebpf:magma_ebpf_kernsnoopd",
+        "//lte/gateway/python/magma/pipelined/ebpf:magma_ebpf_pipelined",
+    ],
+    prefix = "/var/opt/magma/ebpf",
+)
+
+pkg_filegroup(
+    name = "magma_python_scripts",
+    srcs = [
+        "//lte/gateway/python/load_tests:magma_lte_loadtest_scripts",
+        "//lte/gateway/python/scripts:magma_lte_scripts",
+        "//orc8r/gateway/python/scripts:magma_orc8r_scripts",
+    ],
+)
+
+pkg_files(
+    name = "magma_go_binaries",
+    srcs = ["//feg/gateway/services/envoy_controller"],
+    attributes = pkg_attributes(mode = "0755"),
+    prefix = "/usr/local/bin",
+)
+
+pkg_files(
+    name = "magma_c_binaries",
+    srcs = [
+        "//lte/gateway/c/connection_tracker/src:connectiond",
+        "//lte/gateway/c/core:agw_of",
+        "//lte/gateway/c/li_agent/src:liagentd",
+        "//lte/gateway/c/session_manager:sessiond",
+    ],
+    attributes = pkg_attributes(mode = "0755"),
+    prefix = "/usr/local/bin",
+    renames = {"//lte/gateway/c/core:agw_of": "mme"},
+)
+
+pkg_filegroup(
+    name = "magma_python_services",
+    srcs = [
+        "//lte/gateway/python:magma_python_lte_services",
+        "//orc8r/gateway/python:magma_python_orc8r_services",
+    ],
+    prefix = PY_DEST,
+)
+
+pkg_filegroup(
+    name = "magma_configs",
+    srcs = [
+        "//lte/gateway/configs:magma_config_files",
+        "//lte/gateway/configs/templates:magma_lte_config_template_files",
+        "//orc8r/gateway/configs/templates:magma_orc8r_config_template_files",
+    ],
+    prefix = "/etc/magma",
+)
+
+pkg_files(
+    name = "magma_config_stretch_snapshot",
+    srcs = [":stretch_snapshot"],
+    prefix = "/usr/local/share/magma",
+)
+
+pkg_tar(
+    name = "magma_content",
+    srcs = [
+        ":magma_c_binaries",
+        ":magma_config_stretch_snapshot",
+        ":magma_configs",
+        ":magma_ebpf",
+        ":magma_go_binaries",
+        ":magma_python_scripts",
+        ":magma_python_services",
+        ":magma_service_definitions",
+        "//lte/gateway/deploy/roles/magma/files:ansible_configs",
+        "//orc8r/tools/ansible/roles/fluent_bit/files:magma_config_fluent_bit",
+    ],
+    package_file_name = "{fname}.tar".format(fname = MAGMA_FILE_NAME),
+)
+
+pkg_deb(
+    name = "magma_deb_pkg",
```

Comment:
yes

Reply:
Does that also mean that the current sctpd.deb build job in CI is superfluous since we are already building it with `...`?

Reply:
Why would it be "superfluous"?

Reply:
Since we would be building the exact same artifact on the same commit twice in the same CI workflow.

Reply:
ah ok, I don't think so as we build it with `--config=production` - I would say it's superfluous if we solve the current test problem with `--config=production`. Then it would be handled by the matrix job.